terms.html

```diff
@@ -0,0 +1,86 @@
+This page contains both our <a href="#terms">Terms of Service</a> (adapted from the bsd.network ToS) and our <a href="#privacy">Privacy Policy</a>.
+
+<h2 id="terms">chaos.social Terms of Service</h2>
+
+<p>It is our intention that you use this service for personal enjoyment and respectful, friendly interaction. To that end, we hope to foster a welcoming and inclusive environment.</p>
+
+<p>The server is privately owned and open to users voluntarily, not a public space. Users wishing to join this community are expected to act without malice and in good faith. Doing otherwise may lead to removal from the service, independent of whether a user violates any rules outlined below.</p>
+
+<p>The administrators and moderators of this instance are <a href="https://chaos.social/@leah">@leah</a> and <a href="https://chaos.social/@rixx">@rixx</a>.</p>
+
+<p>The server hosting the instance is located in Germany.</p>
+
+<p>The following statements apply regardless of privacy level and instance of the users involved. In rare cases, public or private offline conduct or conduct on a separate instance may constitute grounds for removal from the service.</p>
+
+<hr class="spacer" />
+
+<h3>Policies and Rules</h3>
+
+<p>Our instance is subject to a set of rules governing user behaviour. The rules are always visible on our <a href="https://chaos.social/about/more">about page</a> and in our <a href="https://github.com/chaossocial/about/blob/master/about.html">public repository</a>.</p>
+
+<p>These rules are designed to maintain a friendly and open atmosphere, and to prevent harassment and discrimination. As such, they are a set of guidelines, but by necessity incomplete. Users violating the spirit of these rules will be treated no differently than users violating a specific rule.</p>
+
+<p>Please note that our rules contain a section on best practices, and users who repeatedly and despite warnings disregard these best practices may be seen to be in violation of our rules.</p>
+
+<p>The moderators may remove accounts who spam the instance, or are suspected of camping just to reserve an account name. Violation of the policies and rules may also lead to account removal at the discretion of the moderators.</p>
+
+<h3>Data Access</h3>
+
+<p>Content on this instance should not be used for the purposes of machine learning or other "research" purposes without the explicit consent of the users involved.</p>
+
+<p>Content on this instance beyond this page should not be archived or indexed wholesale by automated means by any user or service. Active users may export their following lists and post content through the web interface.</p>
```

Comment:
"posting content through the web interface" may be to specific. What's about the api and third party clients?

Reply:
Hm, the third-party clients and API don't really contain a complete archive (or they do technically, but it's unusable due to rate limiting). I think the only way to export in a reasonable time is in the settings page.

Changed to

"<p>Content on this instance beyond this page must not be archived or indexed wholesale by automated means by any user or service. Active users may export their following lists and posts through the export provided on their settings page, or the API.</p>"